src/scripts/main.js

```diff
@@ -1,3 +1,210 @@
 'use strict';
 
-// write code here
+const body = document.body;
+const tableBody = document.querySelector('tbody');
+const tableHead = document.querySelector('thead');
```

Comment:
раз уже ты уже нашел body
```suggestion
const body = document.body;
const tableBody = body.querySelector('tbody');
const tableHead = body.querySelector('thead');
```